pkg/api/validation/schema.go

```diff
@@ -88,6 +101,29 @@ func (s *SwaggerSchema) ValidateBytes(data []byte) error {
 	if kind == nil {
 		return fmt.Errorf("kind not set")
 	}
+	if kind.(string) == "List" {
+		items, err := unpackObject(obj.(map[string]interface{}))
+		if err != nil {
+			return err
+		}
+		for _, item := range items {
+			fields := item.(map[string]interface{})
```

Comment:
Are you doing this because of `runtime.Unstructured`?  It would be better to just use `runtime.Unstructured` directly here, to be consistent with the rest of the list handling code.